tools/build/README.md

```diff
@@ -76,3 +89,6 @@ or see [these instructions](https://pypi.python.org/pypi/setuptools/0.9.8#instal
 ```
 pip install --upgrade setuptools
 ```
+
+[1]: https://gradle.com/build-scans
+[2]: https://scans.gradle.com
```

Comment:
We usually inline links with the `[text](link)` notation. Would you mind changing that?

Reply:
Done now